package.json

```diff
@@ -167,7 +167,6 @@
     "node-hid": "2.1.1",
     "numeral": "git+https://github.com/LiskHQ/Numeral-js.git",
     "path-browserify": "^1.0.1",
-    "postcss": "8.4.12",

```

Comment:
@reyraa should we remove the postcss from here?

Reply:
It's already in the resolutions.